Cargo.toml

```diff
@@ -17,9 +17,9 @@ categories = ["science"]
 license = "MIT OR Apache-2.0"
 
 [workspace.dependencies]
-candle-core = { git = "https://github.com/huggingface/candle.git", version = "0.4.0" }
-candle-examples = { git = "https://github.com/huggingface/candle.git", version = "0.4.0" }
-candle-nn = { git = "https://github.com/huggingface/candle.git", version = "0.4.0" }
+candle-core = "0.4.1"
```

Comment:
I prefer to get the latest git version as it provides the latest optimizations and new features. Could you perhaps use that format, ex: `{ git = "https://github.com/huggingface/candle.git", version = "0.4.1" }`